aeron-driver/src/main/c/aeron_windows.cpp

```diff
@@ -0,0 +1,249 @@
+#include "util/aeron_platform.h"
+
+#if defined(AERON_COMPILER_MSVC)
+#include <filesystem>
+#endif
+
+#include <chrono>
+#include <regex>
+#include <thread>
+#include <time.h>
+
+
+extern "C"
```

Comment:
We don't want C++ inside the C media driver. Tests can use C++, but not the base driver.